website/blog/2023-3-01-fonicom.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: fonicom
+title: "How Corso helps Fonicom deliver a more reliable experience for internal backups"
+description: "Corso, an open-source tool for backing up Microsoft 365 Data, was lucky to have Kias Hanifa from Fonicom take a look at our tool. He was kind enough to write this guest post."
+authors: kias
+tags: [corso, microsoft 365, backups, MSP]
+date: 2023-3-01
+image: ./images/servers.png
+---
+
+![media server By Victor Grigas/Wikimedia Foundation, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=44043618](./images/servers.png)
+
+*Corso, an open-source tool for backing up Microsoft 365 Data, was lucky to have
+<!-- vale Vale.Spelling = NO -->
+Kias Hanifa from Fonicom take a look at our tool. He was kind enough to write
+this guest post*
+<!-- vale Vale.Spelling = YES -->
+<!-- truncate -->
+---
+
+I’d like to tell the story of how Corso is helping us build a more reliable experience for our clients.
+
+To start with, a word about my company
+
+Fonicom is a multi-disciplinary ICT solutions company specializing in systems
+integration and software development. Our systems integration expertise covers
+networking, security, virtualisation, mobility, storage, Linux, Open-source
+technologies, public and private cloud solutions, business continuity,
+productivity, unified communications and collaboration, amongst others.
+
+## From open source to public cloud
+
+Originally, our main differentiator was our knowledge of open-source tools. We
+were open-source enthusiasts, and it had a strong business case as one of our
+key offerings. As we’ve grown, and as a SaaS/open-source hybrid approach has
+gotten more standard, we’ve now got AWS Partner, Dell Titanium Partner, Google,
+and Microsoft Partner status.
+
+There was a time when people mainly looked to managed service providers to give
+expertise in tools that had no other support. In the last several years though
+everyone acknowledges the need for public cloud expertise.
+
+As we got into cloud offerings, we found things like Microsoft Office 365, and
+AWS S3, to offer market-leading solutions to our customers. I further identified
+that Microsoft 365 backups were a key area where people wanted better tools.
+This led me to discover Corso.
+
+## The most important part of any open-source project
+
+When I first explored Corso I knew right away that they had the most important
+feature for any open-source project, or any software project: clear
+documentation.
+
+Corso is a tool for bridging two public clouds: taking backups from Microsoft
+Office 365 and sending it to object storage such as AWS S3. That means setup
+<!-- vale alex.Condescending = NO -->
```

Comment:
Again broken when built. You can move the vale stanzas to the beginning of the paragraph

Reply:
fixed